sample-app/shared/src/commonMain/kotlin/org/publicvalue/multiplatform/oidc/sample/home/HomePresenter.kt

```diff
@@ -105,9 +103,20 @@ class HomePresenter(
                     if (client != null) {
                         tokenResponse?.let {
                             scope.launch {
-                                if (!client.config.endpoints?.endSessionEndpoint.isNullOrEmpty()) {
+                                val isGoogle = client.config.discoveryUri.toString().contains("accounts.google.com")
+                                if (!client.config.endpoints?.endSessionEndpoint.isNullOrEmpty() || isGoogle) {
                                     catchErrorMessage {
-                                        val result = client.endSession(idToken = it.id_token ?: "")
+                                        val result = if(isGoogle) {
+                                            val endpoint = "https://accounts.google.com/o/oauth2/revoke"
+                                            val url = URLBuilder(endpoint)
+                                            val response = DefaultHttpClient.submitForm {
+                                                url(url.build())
+                                                parameter("token", it.access_token)
+                                            }
+                                            response.status
+                                        } else {
+                                            client.endSession(idToken = it.id_token ?: "")
```

Comment:
Would this also work using client.endSession() { //configure block to add token parameter } ?